tools/travis.sh

```diff
@@ -22,6 +23,9 @@ update_distro_packages()
     case "${DISTRO}" in
         centos*)
             sudo yum install -y epel-release
+            # remove fake aspcud and install our own rpms
+            sudo rm -rf /usr/bin/aspcud
+            sudo yum install -y tools/aspcud/*.rpm
```

Comment:
Do we need `tools/aspcud/aspcud_xs.tar.gz` too, or can we remove it from the PR?

Reply:
Would it be better to curl/wget the RPMs rather than including them here? I understand that this adds a point of failure but adding architecture-specific RPMs into the repo is also not nice.

Reply:
I left the `tar.gz` as it contains the specs and patches to generate those rpms. I preferred to leave them here to keep things simple and contained, but we can figure out an alternative

Reply:
What if we put them in the xapi CI scripts repo? We only use that to wget ci related stuff after all

Reply:
What if we put them in the xapi CI scripts repo? We only use that to wget ci related stuff after all